deepfence_server/reporters/scan/scan_result_actions.go

```diff
@@ -75,7 +84,12 @@ func UpdateScanResultMasked(ctx context.Context, req *model.ScanResultsMaskReque
         MATCH (s) -[:SCANNED] ->(e)
         MATCH (c:ContainerImage{node_id: e.docker_image_id}) -[:ALIAS] ->(t)
         MERGE (t) -[m:MASKED]->(n)
-        SET m.masked = $value`
+        SET m.masked = $value
+		WITH c,n
+		MATCH (c) -[:IS] ->(ist)
```

Comment:
Do we really need this line, can't we reuse `c` & `t`? or `r`?

Reply:
I had to add this as the query was giving an error saying "WITH is required between SET and MATCH".

Reply:
Yes but is `ist` no the same thing as `t`? (Asking not sure)
If so, we could simply do `WITH c,n,t`

Reply:
`ist` and `t` are different nodes as below:
t -> ImageTag
ist -> ImageStub